src/js/components/Meta.js

```diff
@@ -340,6 +374,25 @@ class Meta extends React.PureComponent {
               )}
             />
           </div>)}
+        
+        {this.state.isShabadPlayable && (
+          <div className={`hukamnama-audio ${(this.state.isHukamnamaAudioPlayerVisible && this.state.isShabadPlayable) ? 'hukamnama-audio--shown' : 'hukamnama-audio--hidden'} ${showPinSettings ? 'hukamnama-audio--pin-settings' : ''}`}>
+          <AudioPlayer
+              ref={this.audioPlayerRef}
+              src={this.state.shabadURL}
+              customAdditionalControls={[]}
+              customVolumeControls={[]}
+              header={(
+                <div>
+                  <h3 className="hukamnama-player-title">{`Listen to this Shabad`}</h3>
```

Comment:
@MySmilingTurban No needs for string interpolation
`Listen to this shabad` without any curly braces and so